examples/js/loaders/FBXLoader.js

```diff
@@ -1651,6 +1651,21 @@
 
 		addAnimations( sceneGraph, animations );
 
+		/*
+			parse ambient color - if it's not set to black (default), create an ambient light
+			Note: no way to set intensity, FBX only provides a color
```

Comment:
You _are_ setting an intensity.

I'd remove this comment and specifically set the intensity to 1.
```javascript
sceneGraph.add( new THREE.AmbientLight( color, 1 ) );
```

Reply:
Fair point, done.